src/client/redux/recipes/selectors.js

```diff
@@ -0,0 +1,3 @@
+const selectContacts = state => state.contacts.items;
```

Comment:
@mattmalicki I recommend you to rename the variable declaring our recipe selector.

Reply:
Right.